src/vldint.h

```diff
@@ -46,6 +47,7 @@
 #include "vldallocator.h"   // Provides internal allocator.
 
 #define MAXMODULELISTLENGTH 512     // Maximum module list length, in characters.
+#define MAXIGNOREFUNCTIONLISTLENGTH 512     // Maximum module list length, in characters.
```

Comment:
>512 [](http://example.com/codeflow?start=36&length=3)

Seems a bit low. In theory this should be dynamic anyway, but at least we should make this somewhat higher

Reply:
Made it to 2048.